cppbuild/cppbuild.cmd

```diff
@@ -57,6 +57,10 @@ if not "%1"=="" (
     ) else if "%1"=="--no-parallel" (
         set "CMAKE_BUILD_PARALLEL_LEVEL=1"
         echo "Disabling parallel build"
+    ) else if "%1"=="--parallel-cpus" (
```

Comment:
I think we shall align Linux and Windows scripts. Currently Linux script defines `--spare-cpu` option which simply leaves one core unused whereas this script adds `--parallel-cpus` option which allows specifying the exact number of cores.

Reply:
I've changed the bash script to use `--parallel-cpus` rather than the other way round.